frontends/mit-open/src/page-components/ChannelDetails/ChannelDetails.tsx

```diff
@@ -91,9 +99,10 @@ const getFacetManifest = (channelType: ChannelTypeEnum) => {
 
       labelFunction: (key: string, channelTitle: string) => (
         // eslint-disable react/jsx-no-target-blank
-        <a target="_blank" href={key} rel="noopener noreferrer">
-          {channelTitle} Website <RiExternalLinkLine size={18} />
-        </a>
+        <ChannelLink target="_blank" href={key} rel="noopener noreferrer">
+          <span>{channelTitle} Website</span>
+          <RiExternalLinkLine size={16} />
```

Comment:
This looks a lot better and the icon and text are the same height now, but because of a combination of factors (line height, internal padding on svg icon) the icon appears to be a bit low compared to the text:

![image](https://github.com/user-attachments/assets/86672aef-4ee9-49be-b7c9-2f1a8e4afd01)

A 2px margin on the bottom of the `RiExternalLinkLine` element tightens it up and really aligns it well:

![image](https://github.com/user-attachments/assets/1a87c2af-e422-4a63-b106-ab39be36ace3)

Reply:
Done!